l10n_it_vat_registries_split_payment/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Bridge module to make VAT registries module work along with Split Payment module
```

Comment:
Come in https://github.com/OCA/l10n-italy/pull/1281#discussion_r291840441, andrebbe creata una descrizione bilingue, vedi le linee guida:
https://www.odoo-italia.org/documentazione/sviluppo/guidelines.html#descrizione-bilingue

Reply:
Fatto!  
Grazie! 😉

Reply:
Grazie a te. :wink: 
Giusto qualche piccolo suggerimento:

- Nel titolo metterei "Scissione dei pagamenti" al plurale, dato che è il termine usato comunemente in italiano.

- in "summary" al posto di modulo "bridge" potresti mettere tranquillamente "modulo ponte", è già usato sia nelle traduzioni che in altri moduli, es. 
https://github.com/OCA/l10n-italy/blob/12.0/l10n_it_fatturapa_out_stamp/readme/DESCRIPTION.rst

- sempre in "summary", se devi indicare nomi specifici di moduli sarebbe meglio metterli tra doppi apici (es. "Registro IVA") altrimenti potresti optare per un più generale (e in minuscolo):
...per il funzionamento congiunto dei registri IVA con la scissione dei pagamenti.